File: plugins/java/Skeleton.java
import java.util.List;
import org.objectweb.asm.tree.ClassNode;
import the.bytecode.club.bytecodeviewer.api.*;

/**
 ** This is a skeleton template for BCV's Java Plugin System
 **
 ** @author [Your Name Goes Here]
 **/

public class Skeleton extends Plugin {

    @Override
    public void execute(List<ClassNode> classNodesList) {
        PluginConsole gui = new PluginConsole("Skeleton Title");
        gui.setVisible(true);
        gui.appendText("executed skeleton example");
    }
}

File: plugins/javascript/Skeleton.js

/**
 ** This is a skeleton template for BCV's Javascript Plugin System
 **
 ** @author [Your Name Goes Here]
 **/

function execute(classNodeList) {
    var PluginConsole = Java.type("the.bytecode.club.bytecodeviewer.api.PluginConsole");
    var gui = new PluginConsole("Skeleton Title");
    gui.setVisible(true);
    gui.appendText("executed skeleton example");
}